docs/help.md

```diff
@@ -6,6 +6,7 @@
 * [Controlling assignment repo settings with the Probot Settings app](/help/probot-settings)
 * [Creating group assignments](/help/create-group-assignments)
 * [Upgrading your organization](/help/upgrade-your-organization)
+* [Connecting to a Learning Management System](/help/connect-to-lms)
```

Comment:
Can we remove this link for now? We can give the teachers a direct link here when user testing, but we shouldn't expose our documentation publicly until launch.